src/IotCentral/IotCentral/help/Set-AzIotCentralApp.md

```diff
@@ -1,93 +1,56 @@
 ---
-external help file: Microsoft.Azure.Commands.IotCentral.dll-Help.xml
-Module Name: AzureRM.IotCentral
-online version: https://docs.microsoft.com/en-us/powershell/module/az.iotcentral/set-aziotcentralapp
+external help file: Microsoft.Azure.PowerShell.Cmdlets.IotCentral.dll-Help.xml
+Module Name: Az.IotCentral
+online version:
 schema: 2.0.0
 ---
 
-# Set-AzureRmIotCentralApp
+# Set-AzIotCentralApp
 
 ## SYNOPSIS
-Updates the metadata for an IoT Central Application.
+{{Fill in the Synopsis}}
 
 ## SYNTAX
 
 ### ResourceIdParameterSet (Default)
 ```
-Set-AzureRmIotCentralApp [-DisplayName <String>] [-Tag <Hashtable>] -ResourceId <String> [-AsJob]
- [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+Set-AzIotCentralApp [-DisplayName <String>] [-Subdomain <String>] [-Tag <Hashtable>] -ResourceId <String>
```

Comment:
I believe we should add `sku` and `companyName` to perform the full functionality of our rp-service update api.

Reply:
@r-delgadillo company name? Also I feel like we intentionally left out Sku. There isn't really any functionality right now--we don't support F1, and you can't update a subless app through the SDK (it's not an azure resource).